src/tests/ftest/util/general_utils.py

```diff
@@ -444,7 +448,7 @@ def get_host_data(hosts, command, text, error, timeout=None):
     data_error = "[ERROR]"
 
     # Find the data for each specified servers
-    log.info("  Obtaining %s data on %s", text, hosts)
+    log.info("  Obtaining %s data on %s using: %s", text, hosts, command)
```

Comment:
This is to compensate for the loss of logging the journalctl command in soak's updated get_journalctl().